backend/schemas/source/sections/GeneralInformation.schema.jsonnet

```diff
@@ -23,9 +23,9 @@ local Types = Base.Types;
     audit_period_covered: {
       '$ref': '#/$defs/AuditPeriod',
     },
-    audit_period_other_months: Types.number{
+    audit_period_other_months: Types.number {
       min: 0,
-      max: 19, #  TODO What should the nax be?
+      max: 19,  //  TODO What should the nax be?
```

Comment:
Just an auto-formatting gremlin.

Reply:
If we don't know what the max should be, we shouldn't constrain it. Could it be 24 (biennial)? More? Less?

Reply:
From the Census docs on this in the past...

```
Item 3 - Audit Period Covered
Mark the appropriate box. Annual audits cover 12 months and
biennial audits cover 24 months. If the audit period covered is
neither annual nor biennial, mark “Other” and provide the number
of months covered in the space provided. The number of months
must be entered as a two-digit number (e.g., for three months
enter ‘03’). See 2 CFR 200.504 for guidelines about the
frequency of audits.
```

https://facides.census.gov/Files/2022%20Checklist%20Instructions%20and%20Form.pdf

Which... doesn't help a lot. 

If we do set min/max, I think they're `minimum`, `maximum`, `exclusiveMinimum`, and `exclusiveMaximum`. 

https://json-schema.org/understanding-json-schema/reference/numeric.html

Reply:
In 2022, it looks like the max was 18 months.

Reply:
Ah. I left my comment on the JSON file. Could you please see that comment. My statement  that 2022 data only had values up to 18 months is not a conclusive answer to what the upper bound in the validation should, or should not, be.